Clarify README: Task notes include descriptions

infotexture opened the pull request:
The full description of the Jira issue is now added to the generated task notes since the changes for #25 in https://github.com/devondragon/jira-omnifocus/commit/7af8765.

This commit updates the README to reflect these changes.

devondragon commented:
@infotexture Thanks for your work on this. However, for my needs I do NOT want the Description synced over, as it is often extremely long and not useful in the context of an OF note.  If you'd be willing to amend this PR to have the Description sync governed by a configuration flag, default of false, I'd be happy to merge it in!

infotexture commented:
@devondragon Thanks for building this. 🙇 

I think there's a misunderstanding here: 

I also _“do NOT want the Description synced over”_ — but the current code on `master` _DOES_ sync the description since your changes for #25 in https://github.com/devondragon/jira-omnifocus/commit/7af8765. 

🔍 See [Line 272](https://github.com/devondragon/jira-omnifocus/blob/master/bin/jiraomnifocus.rb#L272).

(In my local clone, I reverted these changes to restore the previous behavior, which includes only the Jira issue link in the `task_notes` — but those changes are not included here.)

If you look at the changes in this PR, you'll see that I haven't touched the code at all. All I changed is the paragraph in the README that suggests the description would **_not_** be synced — because at the moment it **_is_** synced, and I found that confusing.

> If you'd be willing to amend this PR to have the Description sync governed by a configuration flag, default of false, I'd be happy to merge it in!

That sounds like a good idea, but unfortunately beyond my Ruby skills.

devondragon commented:
Oh man, egg on my face:). Got it!  Okay, I will merge the documentation fix in, and then also create a task to make the description syncing configurable:). Thanks!
